modules/ols-release-notes-known-issues.adoc

```diff
@@ -0,0 +1,12 @@
+// This module is used in the following assemblies:
+// release_notes/ols-release-notes.adoc
+
+:_mod-docs-content-type: Reference
+[id="ols-release-notes-known-issues_{context}"]
+= Known issues
+
+This section captures the issues identified in {ols-long} releases.
+
+== Known issues identified in version 0.1.5
+
+* OLS-757: The {ols-long} image size has grown due to changes associated with the hermetic build process. As a result, the time to obtain the image has increased by a few minutes.
```

Comment:
I can't link to the Jira artifact because the board is not public for the DP program. I know having doc for a DP program is unusual, but the program obtained approval. I sent a message to Kathryn to confirm if the static Jira entry is sufficient in this case.